### ISSUE TITLE
Starting a chroot with xorg breaks dev console ctr-alt refresh (freon)

### ISSUE THREAD

**Issue by divx118** [dnschneid/crouton#1557]
```
chronos@localhost / $ sudo enter-chroot -n trusty croutonversion
Entering /mnt/stateful_partition/crouton/chroots/trusty...
Unknown username "whoopsie" in message bus configuration file
crouton: version 1-20150327195116~master:cf2aacba
release: trusty
architecture: amd64
xmethod: xiwi
targets: xiwi,xorg,xfce,keyboard,extension,chrome
host: version 6885.0.0 (Official Build) dev-channel falco 
kernel: Linux localhost 3.8.11 #1 SMP Mon Mar 16 19:46:02 PDT 2015 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
```
Like the title says, dev console is not reachable anymore with ctr-alt refresh after having a chroot started with Xmethod xorg on devices with freon. Only a reboot gets the situation back to normal. Logging out of the chroot has no effect.

**Comment by dnschneid** [comment]
Yeah, unfortunately we have to kill frecon because [it's currently eating the wrong key combo](https://code.google.com/p/chromium/issues/detail?id=457075); see f11169f4cdb4f6502ef8d023713543b937e80f71.  Once that's fixed we could stop killing it, although we'd have to be sure to disable the crouton keyboard shortcut if frecon is currently active (since there's no way to ask frecon to drop DRI), and we'd have to make sure frecon doesn't asplode if crouton is active and you hit ctrl+alt+F1/F2.

You can relaunch frecon via the command line; iirc `sudo frecon` works.

**Comment by divx118** [comment]
Thanks for the info

**Comment by DennisLfromGA** [comment]
Is https://github.com/dnschneid/crouton/commit/f11169f4cdb4f6502ef8d023713543b937e80f71 still necessary? It looks like a fix for '[frecon responds to Ctrl+Alt+SHIFT+fkey in addition to Ctrl+Alt+fkey](https://code.google.com/p/chromium/issues/detail?id=457075)' may have rolled out in release-R42-6812.B but I'm not sure.

**Comment by dnschneid** [comment]
Yeah, we could revert that, but we need a way to detect that frecon is currently foregrounded and prevent crouton from attempting to switch in that situation.

**Comment by dnschneid** [comment]
This has been fixed with #2097
